scripts/transfer-funds.ts

```diff
@@ -43,10 +43,8 @@ describe("Argent account", function () {
             recipient: recipientAddress,
             amount: { high: 0, low: transferAmount } // works for transferAmount < 2**128
         };
-        const estimatedFee = await sender.estimateFee(token, "transfer", transferArgs);
-        await sender.invoke(token, "transfer", transferArgs, {
-            maxFee: estimatedFee.amount * 2n
-        });
+
+        await sender.invoke(token, "transfer", transferArgs);
```

Comment:
`transferArgs` doesn't have to be defined separately, it can be passed directly now.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-example/pull/79/commits/bdd3f2d24cb951faf4fdeee05a70f1530e93055b